ui/templates/welcome.html

```diff
@@ -29,3 +30,14 @@
         </div>
     {% endif %}
 {% endblock %}
+{% block postcontent %}
+{{ block.super }}
+{% compress js %}
+  <script type="text/javascript"
```

Comment:
Actually it may be better to put the `require(["jquery", "bootstrap"])` in `base.html` instead. Thoughts @carsongee ?